projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -61,4 +62,21 @@ public static function load_coming_soon() {
 			require_once __DIR__ . '/features/coming-soon/coming-soon.php';
 		}
 	}
+
+	/**
+	 * Load WP REST API plugins for wpcom
+	 */
+	public static function load_wpcom_rest_api_endpoints() {
+		if ( ! function_exists( 'wpcom_rest_api_v2_load_plugin_files' ) ) {
+			return;
+		}
+
+		// WPCOM uses ABSPATH in `wpcom_rest_api_v2_load_plugin_files` so we must as well.
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			$relative_path = str_replace( ABSPATH, '', __DIR__ );
+			wpcom_rest_api_v2_load_plugin_files( $relative_path . '/features/wpcom-endpoints/*.php' );
+		} else {
+			wpcom_rest_api_v2_load_plugin_files( 'features/wpcom-endpoints/*.php' );
```

Comment:
Ugh, your change in 8bc07e43 caused me to notice that the non-IS_WPCOM case is wrong too. [It loads relative to the file in Jetpack that defines the function](https://github.com/Automattic/jetpack/blob/8bc07e43e781efb70d2db732b4c0b90f1220875b/projects/plugins/jetpack/_inc/lib/core-api/load-wpcom-endpoints.php#L25), not relative to this file.

Maybe we should just have our own copy of the function here. Neither does anything particularly complicated, they just call `glob()`, filter with `is_file()`, and then `require_once` everything.

Reply:
Oh boy I failed to notice it wasn't loading because my old version of this in Jetpack-the-plugin was still operating on my WoA box `:facepalm:`. I'll roll a custom loader, doesn't need to use the function.

Reply:
@anomiex alright 85ff580facdabdfb6471dac30932006a1b1dd995 should do it, nice and tidy on both platforms.